Doc/library/ssl.rst

```diff
@@ -586,6 +592,42 @@ Constants
    be passed, either to :meth:`SSLContext.load_verify_locations` or as a
    value of the ``ca_certs`` parameter to :func:`wrap_socket`.
 
+.. class:: HostFlags
```

Comment:
Do all of these need to be a public API?

I don't see a need to support things like partial wildcards or multi-label wildcards.

The previous implementation didn't support them, and I don't think expanding the public API like this is a good idea.

Reply:
Partial wildcard matching is disabled by default. I have to expose the flag value so the getter can report the flag correctly:

```
>>> import ssl
>>> ssl.create_default_context().host_flags
<HostFlags.HOSTFLAG_NO_PARTIAL_WILDCARDS: 4>
```

I also like to add ``NEVER_CHECK_SUBJECT`` for urllib3, too.

I don't care about the other flags. I'm happy to remove them.